src/styles/common.scss

```diff
@@ -360,6 +360,43 @@
     }
 }
 
+/* Tabs **************************************************************/
+
+.tabs {
+    display: flex;
+    border-bottom: 1px solid var(--secondary1);
+    overflow: scroll;
```

Comment:
Should be `auto`, I guess. On Windows `overflow: scroll` means there's always a scrollbar, AFAIK.